_layouts/established.html

```diff
@@ -0,0 +1,118 @@
+---
+layout: default
+---
+
+<!-- Pull only the promos that exist for the specific location if any -->
+{% assign location_happenings = site.promos | where: "target_audience", page.name | sort: "date" | reverse %}
+{% assign series = site.series | last %}
+{% include header.html %}
+
+<!-- Resi Player -->
+{% assign locations_with_live_stream =  "oakley, florence, mason, eastside, westside, lexington, uptown" | split: ", "%}
+{% for location in locations_with_live_stream %}
+  {% if page.slug == location %}
+    <section class="bg-blue" id="resi-player">
+      <div class="container">
+        <div class="row text-center push-top">
+          <h2 class="flush-top font-family-condensed-extra font-size-h2 text-uppercase text-white">Welcome to Crossroads {{ page.name }}</h2>
+        </div>
+        <div class="row push-ends">
+          <div class="col-sm-10 col-sm-offset-1">
+            <div id="resi-video-player" data-embed-id="d3ebebe0-b605-48ab-a6e8-b9f3ef201b4e"></div>
+            <script type="application/javascript" src="https://control.resi.io/webplayer/loader.min.js"></script>
+          </div>
+        </div>
+        <div class="row text-center text-white push-bottom">
+          <p class="flush-bottom font-size-small text-uppercase">You're watching the</p>
+          <h3 class="flush-top font-family-condensed-extra font-size-h2 text-uppercase text-white">Crossroads Live Weekend</h3>
+          <a class="text-orange" href="javascript:void(0)" data-smooth-scroll-to="happenings" data-smooth-scroll-offset="50">View events & happenings at your location</a>
+        </div>
+      </div>
+    </section>
+  {% endif %}
+{% endfor %}
+
+<!-- Featured Section -->
+{% if page.featured_section.size > 0 %}
+  <div class="overflow-hidden">
+    {{ page.featured_section }}
+  </div>
+{% endif %}
+
+<!-- Jumbotron section -->
+<section class="jumbotron locations-jumbotron flush-bottom width-100" style="background-image: url('{{ page.bg_image.url | imgix: site.imgix }}?{{ site.imgix_params.placeholder }}')"
+  id="location-page"  
+  data-optimize-bg-img>
+  <div class="bg-overlay"></div>
+  <div class="container">
+    <div class="row location-title-row" style="background-image: url('{{ page.bg_image.url | imgix: site.imgix }}?{{ site.imgix_params.placeholder }}')" data-optimize-bg-img>
+      <div class="bg-overlay"></div>
+      <div class="col-sm-12 push-ends mobile-soft-sides">
+        <div>
+          <span class="font-family-base text-uppercase text-white">
+            <strong>welcome to</strong>
+          </span>
+          <h1 class="flush-ends font-family-condensed-extra text-uppercase">Crossroads <br />{{ page.name }}</h1>
+        </div>
+      </div>
+    </div>
+    <div class="row location-info-row">
+      {% include _location-info.html %}
+    </div>
+  </div>
+</section>
+
+<!-- Featured Section - Below Jumbotron -->
+{% if page.featured_below.size > 0 %}
+  {{ page.featured_below }}
+{% endif %}
+
+
+
+<!-- Modal -->
```

Comment:
Is there a reason why we are not using `_includes/_subscribe-modal.html`? It looks pretty similar but I could be missing something.

Reply:
The hubspot form is different for each location